test/thermodynamics/heat_transfer_equation_test.py

```diff
@@ -0,0 +1,66 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    SI,
+    convert_to
+)
+from symplyphysics.core.symbols.celsius import Celsius, to_kelvin_quantity
+from symplyphysics.laws.thermodynamics import heat_transfer_equation as heat_flow
+
+# What amount of heat is transferred through the wall from one side to the other if the material has a thermal conductivity of 0.4 Watt/(Kelvin*meter), the cross section thickness is 1 cm, its area is 100 cm**2, the temperature inside is 60 Celsius, the temperature outside is 20 Celsius
```

Comment:
Amount of heat transferred is energy, it is measured in joules. And you've got power, which is measured in Watts. Check your example.